js/app.js

```diff
@@ -727,6 +869,18 @@
 
 	// implement function here
 
+	const computeSumWithTax = (taxRate = 8.125, list) => {
+		let total = 0;
+		let preTaxSum = 0;
+		let tax = 0;
+		tax = (taxRate / 100);
+		preTaxSum = computeSum(list);
+		total = (tax * preTaxSum ) + preTaxSum;
+		return total;
+		//Doesnt Pass test but it looks right. computeSum is the function above; 13
```

Comment:
The only thing here is the parameters of `taxRate` and `list` are reversed (I know, it's kind of silly that order matters with named variables but that's the way it works).